specifyweb/frontend/js_src/lib/components/WbPlanView/mappingPreview.ts

```diff
@@ -110,6 +106,7 @@ export function generateMappingPathPreview(
         : (optionLabel as string);
     }),
   ];
+  const isAnyRank = mappingLineData[0].fieldsData['$-any'] !== undefined;
```

Comment:
please don't use `$-any` once again 😢 
there exists `anyTreeRank`

Reply:
There already is a (possibly more robust) check for `any` in this file:
https://github.com/specify/specify7/blob/041259050c024debfa26bef7a0a68df01330dd6d/specifyweb/frontend/js_src/lib/components/WbPlanView/mappingPreview.ts#L175

Can you extract that into a variable and reuse it?